docs/k0sctl-install.md

```diff
@@ -119,29 +133,41 @@ kube-system   kube-proxy-xp9r9                           1/1     Running   0
 kube-system   metrics-server-6fbcd86f7b-5frtn            1/1     Running   0          3m51s
 ```
 
-### Upgrade a k0s cluster using k0sctl
+## Upgrade a k0s cluster using k0sctl
+
+The upgrading of k0s clusters occurs not through a particular command (there is
+no ``upgrade`` sub-command in k0sctl) but by way of the configuration file. The
+configuration file describes the desired state of the cluster. and when that
+description passes to the `k0sctl apply` command a discovery of the current
```

Comment:
it has to be passed, it doesn't pass on it's own :) 

Also something went wrong with the dot `.`.

Reply:
Now reads: "...cluster, and when you pass the description to the `k0sctl apply` command..."